api/gradlew

```diff
@@ -36,9 +36,9 @@ while [ -h "$PRG" ] ; do
     fi
 done
 SAVED="`pwd`"
-cd "`dirname \"$PRG\"`/" >/dev/null
+cd "`dirname \"$PRG\"`/" >/dev/null || exit
```

Comment:
we can use $() syntax as well

Reply:
Updated all the usages of ` in gradlew to use $()